### ISSUE TITLE
Do vendor review for shareprogress.org

### ISSUE THREAD

**Issue by valianttry** [MozillaFoundation/encryption_campaign#24]
Evaluate shareprogress.org to use as "Optimizely for social"

@HPaulJohnson has had a conversation with CEO Jim Pugh to launch discussions.

Outstanding questions:
- Does this tool provide additional functionality that existing tools do not provide (e.g. Sprinklr)?
- How would this tool help us reach our KPIs / goals?
- How does SP use cookies or other tracking?
- If we move forward, does this tool require security and vendor review?

cc @adamlofting @shaghdoosti @KevZawacki

**Comment by adamlofting** [comment]
Looks pretty cool. :+1: 

> Does this tool provide additional functionality that existing tools do not provide (e.g. Sprinklr)?

I don't know Sprinklr well, but believe it's focused on the content that appears directly on social properties (e.g. the content of mozilla account tweets). Share Progress seems to be about optimizing the share ask as it appears on our web pages, and in turn the messaging that supporters tweet (for example) on our behalf.

> How would this tool help us reach our KPIs / goals?

Need to agree the goals first :wink: but if we have a social goal, it would help, if it works. Given the data aspect of this it should be easy to evaluate if it was valuable after the campaign. We should get a quote and see if it seems worthwhile in relation to our budget/goals. Their consulting services could be valuable to the campaign too (again subject to budget)

> How does SP use cookies or other tracking?

This would be checked in the vendor review.

> If we move forward, does this tool require security and vendor review?

Yes - Vendor review to include Legal / Data / Security

**Comment by HPaulJohnson** [comment]
Sprnklr is used primarily for social monitoring/listening. It does some have some extended functionality around buying paid social media placements but that's different from what we are discussing here (share optimization). 

Pricing is at the bottom of this page http://www.shareprogress.org/tools/. Sliding scale based on numbers of shares(ex. $500 mo for 20K shares, first month free as part of a trial. 

Setting up a meeting with Jim next week. If you wanted to be invited let me know.

**Comment by OpenMatt** [comment]
- @HPaulJohnson are you still the right assignee / driver for this ticket? 
- what's the next step here? Anything we need to get done on it this week specifically?

**Comment by HPaulJohnson** [comment]
@OpenMatt you can keep me as driver for this ticket. 

Finding a time for @valiantry and @shaghdoosti to meet with Jim Pugh from Shared Progress this week to make a decision about implementing for this campaign.

**Comment by HPaulJohnson** [comment]
@valianttry, @shaghdoosti and I had a good meeting with Share Progress team yesterday. They gave us a demo and answered all our initial questions. We agree that it would be a good fit for this campaign so next step is moving forward with a vendor review, which I've asked @davidascher for help with.

**Comment by OpenMatt** [comment]
ok great. re-titled ticket to focus on vendor review. rolling milestone over to next week.

**Comment by HPaulJohnson** [comment]
Discussed with @davidascher and per his suggestion emailed Abigail to initiate process. 

Also, meeting with @lovegushwa and @natalieworth on Monday to demo tool so they can better understand how it integrates into landing page.

**Comment by OpenMatt** [comment]
ok thanks @HPaulJohnson -- please let me know if you run into any blockers re: getting vendor review finalized. Will leave with you to push this one to close.

**Comment by HPaulJohnson** [comment]
Questionnaire sent to Share Progress. Also looped @simonwex for help on vendor review. 

Previewed tool for @lovegushwa and @natalieworth who will now explore customization options in a sandbox account that has been setup and compile a list of outstanding to ask Share Progress team.

**Comment by HPaulJohnson** [comment]
Bug filed and waiting for feedback. Moving to next milestone.

https://bugzilla.mozilla.org/show_bug.cgi?id=1243023

**Comment by OpenMatt** [comment]
@simonwex can you help us expedite this at all?

**Comment by HPaulJohnson** [comment]
Update: answered initial set of questions from Marshall; he has a few follow up questions and he, Jim (from Share Progress) and I have a meeting Friday morning to review.

**Comment by HPaulJohnson** [comment]
Update: Met with Marshall Erwin, Jonathan Cladius and Jim Pugh this morning. Most questions answered but Jonathan has requested Share Progress complete an updated, more comprehensive security review questionnaire and review results of that with Simon and I on Tuesday. New questionnaire sent to Jim and meeting set for Tuesday but... 

For microsite launch we should just use our standard share buttons. We will need to incorporate/update with Share Progress post-launch when the review process concludes.

**Comment by OpenMatt** [comment]
@HPaulJohnson let us know when there's any news here.

**Comment by HPaulJohnson** [comment]
Vendor and security review complete this week. 

Share Progress needs to recognized DNT prior to launch. They are working on it now and I'm meeting with them on Monday to understand when that work will be complete. 

In parallel, will work with Abigail on the contract piece (she was waiting for security review to be completed).

**Comment by HPaulJohnson** [comment]
Almost done. 

Share Progress has made necessary changes on their side. Abigail is drafting a special adendum to their TOS that reflects these changes that Share Progress will need to sign. Once that happens, I'll close this and we can move tom implementation.

**Comment by HPaulJohnson** [comment]
Agreement signed. This part is done, now we can move to implementation. Setting up a separate ticket for that.
